aqi_monitor/apps/monitor/tasks.py

```diff
@@ -17,6 +20,16 @@
 def get_aqi():
     region = "saratov"
     response = requests.get(
-        "https://api.waqi.info/feed/" + region + "/?token=7162b8553552c91543ed0ae44c44e59b3724a08e")
-    result = json.loads(response.text)
-    print(result)
\ No newline at end of file
+        f"https://api.waqi.info/feed/{region}/?token=7162b8553552c91543ed0ae44c44e59b3724a08e")
+    data = json.loads(response.text)["data"]
+    idx = {"id_region": data["idx"]}
+    aqi_val = {"aqi": data["aqi"]}
+    aqi = fun.parse_iaqi(data["iaqi"])
```

Comment:
Do we really need to parse the response data here? I suppose you have the special function for that purpose.